### PR TITLE
add a manual test experiment.

### PR DESCRIPTION
**Pull request by kdub**
Add a manual test experiment to facilitate running experimental
tests that are either tricky to script, or used as an initial
data collection to test out a theory.

### CODE REVIEW

File: libs/utils/android/workloads/manual.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        self.collect = collect
+
+        # Set min brightness
+        Screen.set_brightness(self._target, auto=False, percent=0)
+
+        # Set timeout to min value
+        Screen.set_timeout(self._target, seconds=0)
+
+        # Prevent screen from dozing
+        Screen.set_doze_always_on(self._target, on=False)
```
Comment (valschneider) [inline libs/utils/android/workloads/manual.py]: `Screen` doesn't have that method... Yet :)

File: libs/utils/android/workloads/manual.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        # Turn on airplane mode
+        System.set_airplane_mode(self._target, on=True)
+
+        # Unlock device screen (assume no password required)
+        Screen.unlock(self._target)
+
+        self.tracingStart(screen_always_on=False)
+        raw_input("Tracing and energy collection started.\n\
+                   Press ENTER when the manual experiment is concluded.")
+        print("collecting logs...")
```
Comment (valschneider) [inline libs/utils/android/workloads/manual.py]: `self._log.info()` would be better

File: libs/utils/android/workloads/manual.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+from android import Screen, System
+from android.workload import Workload
+
+
+class Manual(Workload):
+    """
+    Android Manual workload
+
+    This workload will set up a device, start tracing, start collection,
+    disconnect USB, and pause. User can then conduct a manual experiment,
```
Comment (valschneider) [inline libs/utils/android/workloads/manual.py]: I don't see code related to handling the USB.

**Comment by derkling** [comment]
@kdub do you have any chance to work on the feedback comments from @valschneider?

We are in the middle of a "merge window"... meaning we allocated a bit more time this week to get as much as possible PR merged into LISA. Thus, would be great if you can update this one, and other from yours, to have a good chance to merge it soon.

**Comment by derkling** [comment]
Closed by error, let's reopen it.

**Comment by valschneider** [comment]
I think it's still a good wload to have. It'd probably be nice to have as a script, so that you can do something like:

```
./manual_experiment --kind android --device DEADBEEF
<play with the device>
ENTER
```

I'll assign that to myself to get it in LISA/next.

**Pull request closed without merging**